diff --git a/src/xrt/ipc/server/ipc_server_handler.c b/src/xrt/ipc/server/ipc_server_handler.c
index 7231ab489..17e79ba8a 100644
--- a/src/xrt/ipc/server/ipc_server_handler.c
+++ b/src/xrt/ipc/server/ipc_server_handler.c
@@ -233,6 +233,7 @@ send_device_inputs(volatile struct ipc_client_state *ics, struct xrt_device *xde
 		// Send the full input state
 		xret = ipc_send(imc, xdev->inputs, input_size);
 		IPC_CHK_ALWAYS_RET(ics->server, xret, "ipc_send(inputs)");
+		return XRT_SUCCESS;
 	}
 
 	struct xrt_input filtered[1024];
@@ -247,7 +248,7 @@ send_device_inputs(volatile struct ipc_client_state *ics, struct xrt_device *xde
 		bool blocked = false;
 		switch (XRT_GET_INPUT_TYPE(src->name)) {
 		case XRT_INPUT_TYPE_POSE: //
-			blocked = iob->block_poses && dst->name != XRT_INPUT_GENERIC_HEAD_POSE;
+			blocked = iob->block_poses && src->name != XRT_INPUT_GENERIC_HEAD_POSE;
 			break;
 		case XRT_INPUT_TYPE_HAND_TRACKING: //
 			blocked = iob->block_hand_tracking;
