diff --git a/src/xrt/targets/common/target_lists.c b/src/xrt/targets/common/target_lists.c
index 929e218..53cb66e 100644
--- a/src/xrt/targets/common/target_lists.c
+++ b/src/xrt/targets/common/target_lists.c
@@ -100,6 +100,10 @@ xrt_builder_create_func_t target_builder_list[] = {
     rift_s_builder_create,
 #endif // XRT_BUILD_DRIVER_RIFT_S
 
+#ifdef T_BUILDER_STEAMVR
+    t_builder_steamvr_create,
+#endif // T_BUILDER_STEAMVR
+
 #ifdef XRT_BUILD_DRIVER_PSVR2
     t_builder_psvr2_create,
 #endif // XRT_BUILD_DRIVER_PSVR2
@@ -112,10 +116,6 @@ xrt_builder_create_func_t target_builder_list[] = {
     t_builder_simula_create,
 #endif // T_BUILDER_SIMULAVR
 
-#ifdef T_BUILDER_STEAMVR
-    t_builder_steamvr_create,
-#endif // T_BUILDER_STEAMVR
-
 #ifdef T_BUILDER_LIGHTHOUSE
     t_builder_lighthouse_create,
 #endif // T_BUILDER_LIGHTHOUSE
